metrics/network/network-metrics-iperf3.sh

```diff
@@ -230,6 +230,34 @@ function parse_iperf_bwd() {
 
 }
 
+# This function parses the output of iperf UDP execution, and
+# saves the receiver successful datagram value in the results.
+# It expects to get a 'clean' JSON formatted input, as that
+# seems to be the 'stable interface' for iperf3 - the UDP output
+# in plain text otherwise can vary between client and host versions
+# which makes parsing it a pain (aka. broken).
```

Comment:
There's an extraneous period after "aka", but I can live with that.. /me twitches ... :smile: